lib/src/extensions/alert_dialog_extensions.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter/material.dart';
+
+extension AlertDialogWithdisableBackButton on AlertDialog {
```

Comment:
@AbdOoSaed This is a very good PR and looks like a nice addition to this package. My only concern is with this new extension, which was introduced in [Dart 2.7](https://dart.dev/guides/language/extension-methods). Would it be possible to add the `disableBackButton` function directly to `AlertDialog` to avoid the extension?